libraries/standard/http/src/http_client.c

```diff
@@ -484,7 +557,7 @@ HTTPStatus_t HTTPClient_InitializeRequestHeaders( HTTPRequestHeaders_t * pReques
     {
         /* Write "User-Agent: <Value>". */
         returnStatus = addHeader( pRequestHeaders,
-                                  ( const uint8_t * ) HTTP_USER_AGENT_FIELD,
+                                  ( const uint8_t * ) ( HTTP_USER_AGENT_FIELD ),
```

Comment:
Why is `HTTP_USER_AGENT_FIELD` wrapped in parenthesis but `HTTP_USER_AGENT_VALUE` is not?

Reply:
No reason oversight thanks